ghost/i18n/locales/hi/ghost.json

```diff
@@ -0,0 +1,26 @@
+{
+    "All the best!": "shubhakaamanaen!",
+    "Complete your sign up to {{siteTitle}}!": "{{siteTitle}} ko Sign up pura karen",
+    "Confirm your email update for {{siteTitle}}!": "Apni email {{siteTitle}} ke liye pura keren",
+    "Confirm your subscription to {{siteTitle}}": "apni subscription {{siteTitle}} ke liye pura karen",
+    "For your security, the link will expire in 24 hours time.": "aapakee suraksha ke lie, link 24 ghanton mein samaapt ho jaega",
+    "Hey there,": "Namaste",
+    "If you did not make this request, you can safely ignore this email.": "yadi aapane yah anurodh nahin kiya hai, to aap is eemel ko surakshit roop se anadekha kar sakate hain",
+    "If you did not make this request, you can simply delete this message.": "yadi aapane yah anurodh nahin kiya hai, to aap is sandesh ko aasaanee se hata sakate hain.",
+    "Please confirm your email address with this link:": "krpaya is link ke saath apane eemel pate kee pushti karen",
+    "Secure sign in link for {{siteTitle}}": "{{siteTitle}} ke lie surakshit sain in link",
+    "See you soon!": "jald hee phir milenge!",
+    "Sent to {{email}}": "{{email}} ko bhej diya",
+    "Sign in to {{siteTitle}}": "{{siteTitle}} ko sign in keejiye",
+    "Tap the link below to complete the signup process for {{siteTitle}}, and be automatically signed in:": "{{siteTitle}} ke lie sainap prakriya ko poora karane ke lie neeche die gae link par taip karen aur apane aap sain in ho jaen",
+    "Thank you for signing up to {{siteTitle}}!": "{{siteTitle}} mein sain ap karane ke lie dhanyavaad!",
+    "Thank you for subscribing to {{siteTitle}}!": "{{siteTitle}} kee sadasyata lene ke lie dhanyavaad!",
+    "Thank you for subscribing to {{siteTitle}}. Tap the link below to be automatically signed in:": "{{siteTitle}} kee sadasyata lene ke lie dhanyavaad. apane aap sain in hone ke lie neeche die gae link par taip karen:",
+    "This email address will not be used.": "is eemel pate ka upayog nahin kiya jaega.",
+    "Welcome back to {{siteTitle}}!": "{{siteTitle}} mein vaapas svaagat hai!",
+    "Welcome back! Use this link to securely sign in to your {{siteTitle}} account:": "vaapasee par svaagat hai! apane {{siteTitle}} khaate mein surakshit roop se sain in karane ke lie is link ka upayog karen:",
+    "You can also copy & paste this URL into your browser:": "aap is url ko apane brauzar mein kopee aur pest bhee kar sakate hain:",
```

Comment:
"Aap is url ko apane ~~brauzar~~ browser mein ~~kopee~~ copy aur ~~pest~~ paste bhee kar sakate hain:"